src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -159,36 +183,30 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
      * it has an example text for the instructor and a default title containing the exercise title
      **/
     createEmptyPost(): void {
-        this.createdPost = this.metisService.createEmptyPostForContext(undefined, undefined, undefined, this.plagiarismCase);
-        this.createdPost.content = `Dear ${this.plagiarismCase.student!.name},\nAfter a meticulous review of your final submission for the ${
-            this.plagiarismCase.exercise!.title
-        } in the course ${
-            this.plagiarismCase.exercise!.course!.title
-        }, we have concluded that you have committed plagiarism.\nThis is not only a violation of principles of good student practice but also of the “Student Code of Conduct” of the faculty of computer science that you have agreed upon. You can check the Student conduct code [here](https://www.in.tum.de/fileadmin/w00bws/in/2.Fur_Studierende/Pruefungen_und_Formalitaeten/1.Gute_studentische_Praxis/englisch/leitfaden-en_2016Jun22.pdf) §22.1 of the “Allgemeine Studien- und Prüfungsordnung (APSO)” [“General Examination and Study Regulations”] regulates consequences for such cases. You can find the APSO [here](https://www.tum.de/studium/im-studium/das-studium-organisieren/satzungen-ordnungen#statute;t:Allgemeine%20Prüfungs-%20und%20Studienordnung;sort:106;page:1).\nYou have one week to provide a statement about this situation.`;
-        this.createdPost.title = `Plagiarism Case ${this.plagiarismCase.exercise!.title}`;
-    }
+        const studentName = this.plagiarismCase.student!.name;
+        const courseTitle =
+            this.plagiarismCase.exercise!.course!.title!.length > 40
+                ? this.plagiarismCase.exercise!.course!.title?.slice(0, 35) + '…'
```

Comment:
Since this is a rather formal text (at least the template is) and the template is long even without the course and exercise titles (~805 chars), I am not sure cropping the course and exercise titles are appropriate.

If the concern here is the 1000 char limit for a post, then I would suggest increasing the limit to somewhere near 90-100 chars because I think 40 char limit is rather low for accommodating full title of many courses (e.g. full title of EIST is 56 chars long: "Introduction to Software Engineering (IN0006) - Garching").

Reply:
The example you mentioned is already quite long, I therefore set the limit to 65 characters.